lib/tasks/ui.rake

```diff
@@ -0,0 +1,15 @@
+require "fileutils"
+
+namespace(:ui) do
+  task(:build) do
+    original_dir = FileUtils.pwd
+
+    begin
+      FileUtils.cd("agile-direction-ui")
+      system("npm install")
```

Comment:
It's [part of the postinstall](https://github.com/agile-direction/agile-direction-ui/blob/master/package.json#L14).